--- a/init.d/Makefile.am	2013-02-14 17:30:46.000000000 +0100
+++ b/init.d/Makefile.am	2013-12-10 16:47:43.000000000 +0100
@@ -30,7 +30,7 @@
 install-data-hook:
 if ENABLE_SYSTEMD
 ## new style init - requires pkg-config.
-	install -p -m644 haveged.service `pkg-config --variable=systemdsystemunitdir systemd`/haveged.service;
+##	install -p -m644 haveged.service `pkg-config --variable=systemdsystemunitdir systemd`/haveged.service;
 ##	systemctl enable haveged.service;
 
 endif
